libraries/AP_Scripting/AP_Scripting_config.h

```diff
@@ -9,7 +9,8 @@
 
 #if AP_SCRIPTING_ENABLED
     #include <AP_Filesystem/AP_Filesystem_config.h>
-    #if !AP_FILESYSTEM_FILE_READING_ENABLED
+    // enumerate all of the possible places we can read a script from.
+    #if !AP_FILESYSTEM_POSIX_ENABLED && !AP_FILESYSTEM_FATFS_ENABLED && !AP_FILESYSTEM_ESP32_ENABLED && !AP_FILESYSTEM_ROMFS_ENABLED
```

Comment:
```suggestion
    #if !AP_FILESYSTEM_POSIX_ENABLED && !AP_FILESYSTEM_FATFS_ENABLED && !AP_FILESYSTEM_ESP32_ENABLED && !defined(HAL_HAVE_AP_ROMFS_EMBEDDED_LUA)
```
I think you can swap the romps one for the romfs with scripts define.

Reply:
Although the existing error string might be a bit critic in that case.

Reply:
... also, I'm allergic to *more* `defined(x)` crap :-)